docker-neo4j/azuredeploy.json

```diff
@@ -0,0 +1,399 @@
+                    "Standard_LRS",
+                    "Standard_GRS",
+                    "Standard_RAGRS",
+                    "Premium_LRS"
+                ],
+                "metadata": {
+                    "description": "The type of the Storage Account created"
+                },
+                "defaultValue": "Standard_LRS"
+            },
+            "networkSecurityGroupName": {
+                "type": "string",
+                "metadata": {
+                    "description": "Name of the network security group."
+                },
+                "defaultValue": "myNSG"
+            },
+            "networkInterfaceName": {
+                "type": "string",
+                "metadata": {
+                    "description": "Name of the network security group."
+                },
+                "defaultValue": "myVMNic"
+            },
+            "publicIPAddressName": {
+                "type": "string",
+                "metadata": {
+                    "description": "Name of the public IP address"
+                },
+                "defaultValue": "myPublicIP"
+            }
+        },
+        "variables": {
+            "imagePublisher": "Canonical",
+            "imageOffer": "UbuntuServer",
+            "ubuntuOSVersion": "15.04",    
+            "OSDiskName": "osDisk",
+            "nsgName": "[parameters('networkSecurityGroupName')]",
+            "nicName": "[concat(parameters('vmName'),parameters('networkInterfaceName'))]",
+            "dockerExtensionName": "DockerExtension",
+            "azureVMUtilsName": "vmCustomizationExt",
+            "azureVMUtilsScriptUrl": "https://github.com/Azure/docker-neo4j/raw/master/extras/setup_data_disk.sh",
+            "addressPrefix": "10.0.0.0/16",
+            "subnetName": "Subnet",
+            "subnetPrefix": "10.0.0.0/24",
+            "storageAccountType": "Standard_LRS",
+            "publicIPAddressName": "[concat(parameters('vmName'),parameters('publicIPAddressName'))]",
+            "publicIPAddressType": "Dynamic",
+            "vmStorageAccountContainerName": "vhds",
+            "vmName": "[parameters('vmName')]",
+            "vmSize": "[parameters('vmSize')]",
+            "virtualNetworkName": "MyVNET",
+            "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+            "nsgID": "[resourceId('Microsoft.Network/networkSecurityGroups',variables('nsgName'))]",
+            "subnetRef": "[concat(variables('vnetID'),'/subnets/',variables('subnetName'))]",
+            "dataDisk1VhdName": "[concat('http://',parameters('newStorageAccountName'),'.blob.core.windows.net/',variables('vmStorageAccountContainerName'),'/',variables('vmName'),'dataDisk1.vhd')]"
+
+        },
+        "resources": [
+        {
+            "type": "Microsoft.Storage/storageAccounts",
+            "name": "[parameters('newStorageAccountName')]",
+            "apiVersion": "2015-05-01-preview",
```

Comment:
Can you make api-version a variable as well so it's easier to move to newer API headers.